website/content/docs/extending-waypoint/plugin-interfaces/platform-releaser.mdx

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+page_title: 'PlatformReleaser'
+description: |-
+  How to implement the PlatformReleaser component for a Waypoint plugin
+---
+
+# PlatformReleaser
+
+https://pkg.go.dev/github.com/hashicorp/waypoint-plugin-sdk/component#PlatformReleaser
+
+The PlatformReleaser interface is an optional interface that a Platform component
```

Comment:
```suggestion
The `PlatformReleaser` interface is an optional interface that a Platform component
```

I think interface names like this should have the special font applied. I know we aren't very consistent here, but we should try to be IMO